### PR TITLE
Resolved Issue #3 Title

### PR DESCRIPTION
**Pull request by Bamboworld**
Resolved title by Title Service and setting it using environment.title

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,9 +3,10 @@
 
 <head>
   <meta charset="utf-8">
-  <title>PracLearn</title>
-  <base href="/">
-
+  <!--Issue #3 Remove <title> <title>PracLearn</title> -->
```
Comment (AlexZhidkov) [inline src/index.html]: Please just delete this line. Commenting code out is a bad practice, it creates "zombie code".

File: src/app/app.component.ts [hunk 2]
```diff
@@ -17,8 +18,13 @@ export class AppComponent {
   photoURL: string;
 
   constructor(public authService: AuthService,
-              private router: Router) {
+              private router: Router,
+              private titleService: Title) {
     this.title = environment.title;
+    
+    // Issue #3 Resolved: Set Title using environment.title
```
Comment (AlexZhidkov) [inline src/app/app.component.ts]: Please delete this comment. Line above doesn't require comment to understand what is happening here. Resolved issue is captured in your PR. We don't want useless comments. Code should be self-documented. Have a look at "clean code" practice.

File: src/index.html (hunk 1 above)
Comment (Bamboworld) [reply src/index.html]: OK, sure.

File: src/app/app.component.ts (hunk 2 above)
Comment (Bamboworld) [reply src/app/app.component.ts]: OK, sure.

**Pull request merged**